go/obscuronode/enclave/enclave.go

```diff
@@ -231,9 +231,10 @@ func (e *enclaveImpl) start(block types.Block) {
 
 func (e *enclaveImpl) ProduceGenesis(blkHash common.Hash) nodecommon.BlockSubmissionResponse {
 	rolGenesis := obscurocore.NewRollup(blkHash, nil, obscurocommon.L2GenesisHeight, common.HexToAddress("0x0"), []*nodecommon.L2Tx{}, []nodecommon.Withdrawal{}, obscurocommon.GenerateNonce(), common.BigToHash(big.NewInt(0)))
+	nodecommon.LogWithID(e.nodeShortID, "Fetching blockHash = %x", blkHash)
```

Comment:
@BedrockSquirrel Looks like leftover debugging code? Why are we logging this block fetch in particular? The error message below is enough.